assets/js/controllers/send.controller.js

```diff
@@ -179,6 +186,11 @@ function SendCtrl ($scope, $log, Wallet, Alerts, currency, $uibModal, $uibModalI
       .then(transactionSucceeded).catch(transactionFailed);
   };
 
+  $scope.sendInputMetrics = (metric) => {
+    let root = $rootScope.rootURL ? $rootScope.rootURL : '/';
+    $http.get(`${root}event?name=wallet_web_tx_from_${metric}`);
```

Comment:
Is this the same analytics endpoint as we use to track 2nd password usage? If so, can you move this to My-Wallet-V3?